ee-feature-pack/galleon-common/src/main/resources/layers/standalone/jaxrs/layer-spec.xml

```diff
@@ -4,4 +4,8 @@
         <layer name="web-server"/>
     </dependencies>
     <feature-group name="jaxrs"/>
+    <packages>
+        <!-- Optional tracing API -->
+        <package name="org.jboss.resteasy.resteasy-tracing-api" optional="true"/>
+    </packages>
```

Comment:
I don't understand why this would be required. If it is we should add a comment as to why it's required.

Reply:
@jamezp I forwarded you about the email thread we discussed about this.

Reply:
Okay. I don't think we need this IIRC.

Reply:
okay!